### ISSUE TITLE
Publish results to triple store does not work

### ISSUE THREAD

**Issue by wouterbeek** [pldn/LDWizard#143]
### Observation

When I enter an API Token under "Publish results to TriplyDB" nothing happens in the GUI, and I receive the following error message in the developer console:

```
Uncaught (in promise) TypeError: _triply_triplydb_ignored_default(...).get is not a function
```

### Expected

Uploading to a triple store works.

**Comment by wouterbeek** [comment]
We looked into this within a timebox, and found the following:
- The issue originates because TriplyDB.js uses a Node.js library (filesystem 'fs-extra'), which cannot be used in a web browser. (This error is currently emitted by Webpack, which checks whether all used libraries are able to run in a web browser. Even if Webpack would not be used, then there would still be an error, but this error would be emitted in the web browser.)

**Comment by philipperenzen** [comment]
Fixed in LDWizard version 2.3.4

See PR #147
